src/p2p/net_node.inl

```diff
@@ -441,6 +442,36 @@ namespace nodetool
         return false;
     }
 
+    if (!command_line::is_arg_defaulted(vm, arg_ban_list))
+    {
+      const std::string ban_list = command_line::get_arg(vm, arg_ban_list);
+
+      const boost::filesystem::path ban_list_path(ban_list);
+      boost::system::error_code ec;
+      if (!boost::filesystem::exists(ban_list_path, ec))
+      {
+        throw std::runtime_error("Can't find ban list file " + ban_list);
```

Comment:
Are these printed ? If not, it should have a MERROR somewhere. Or MFATAL since it appears to die. Adding the ec message to the log would also be nice.

Reply:
Added ec message. Looks like this:

```
selsta@mbpR ~/d/m/b/D/b/r/bin (ban-list)> ./monerod --ban-list ~/.bitmonero/block.tx2t
2020-10-20 17:55:07.425	I Monero 'Oxygen Orion' (v0.17.0.0-aad2f87a5)
2020-10-20 17:55:07.425	I Initializing cryptonote protocol...
2020-10-20 17:55:07.425	I Cryptonote protocol initialized OK
2020-10-20 17:55:07.425	I Initializing core...
2020-10-20 17:55:07.426	I Loading blockchain from folder /Users/selsta/.bitmonero/lmdb ...
2020-10-20 17:55:07.503	I Loading checkpoints
2020-10-20 17:55:07.504	I Core initialized OK
2020-10-20 17:55:07.504	I Initializing p2p server...
2020-10-20 17:55:07.504	I Deinitializing core...
2020-10-20 17:55:07.517	I Stopping cryptonote protocol...
2020-10-20 17:55:07.517	I Cryptonote protocol stopped successfully
2020-10-20 17:55:07.517	E Exception in main! Can't find ban list file /Users/selsta/.bitmonero/block.tx2t - No such file or directory
```